### DISCUSSION TITLE
Installing Postgis

### DISCUSSION THREAD

**Discussion by manuelcalerosolis** [Tecnativa/docker-postgres-autoconf#10] [Q&A]
I am trying to install the library to use Postgis within Posgres but I don't know how I can do it using docker-postgres-autoconf, to be able to use the repository [https://github.com/OCA/geospatial](https://github.com/OCA/geospatial)

Thanks.

**Comment by pleirb** [comment]
> I am trying to install the library to use Postgis within Posgres but I don't know how I can do it using docker-postgres-autoconf, to be able to use the repository https://github.com/OCA/geospatial
> 
> Thanks.

I'm in the same situation. You're probably using the Doodba copier template and don't want to alter the YAML files the copier generates.

One solution would be to add a Dockerfile that runs `RUN apk add --no-cache postgis`, but then we risk overwriting these files if we update with the copier.

Does anyone know the correct way to do this?

For now, I'm testing with the Siplo image, which already includes PostGIS, but again, it's not Doodba's native solution.

**Reply by pedrobaeza** [reply]
Why adding it to `apt.txt` or `apt_build.txt` doesn't serve?

**Reply by pleirb** [reply]
> Why adding it to `apt.txt` or `apt_build.txt` doesn't serve?

I understand that `apt.txt` and `apt_build.txt` only affect the Odoo service, not the database service, which is where PostGIS needs to be installed.

**Reply by pedrobaeza** [reply]
@david-banon-tecnativa can you check this?
